tests/marker_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests_test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/ovs-cni/tests"
+)
+
+var _ = Describe("ovs-cni-marker", func() {
+	Describe("bridge resource reporting", func() {
+
+		Context("when bridge is configured on a node", func() {
```

Comment:
@booxter @SchSeba could you guys please help me here? When I run this context or the second one alone, it passes. However, when I run them together, it fails saying that the resource is not reported.

I made sure that bridge is indeed created. However, it seems like it is removed right after. Maybe both contexts run in parallel?